keep/api/models/db/migrations/versions/2024-08-14-18-30_87594ea6d308.py

```diff
@@ -0,0 +1,57 @@
+"""add rules-related fields to the incident
```

Comment:
wouldn't it make sense to have one migration per PR?

Reply:
Going to squash them